### ISSUE TITLE
Correct formatting for user-defined arrays

### ISSUE THREAD

**Issue by metthal** [avast/yaramod#171]
As pointed out in #170, autoformatting of user-defined arrays is not handled completely right because it doesn't indent the inner content of the enumeration of the expressions. Please make it so that these user-defined arrays are formatted like this

```yara
	condition:
		1 of [
			cuckoo.sync.mutex(/a/),
			cuckoo.sync.mutex(/b/)
		]
```

**Comment by TadeasKucera** [comment]
Thank you Marek for the fix in #170. I also fixed the incorrect indenting within the [] brackets in #172 .

**Comment by TadeasKucera** [comment]
Resolved in #170 and #172
